src/test/java/com/digitalasset/testing/junit4/SandboxTest.java

```diff
@@ -0,0 +1,30 @@
+package com.digitalasset.testing.junit4;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExternalResource;
+
+import java.nio.file.Paths;
+
+public class SandboxTest {
+
+  private static Sandbox sandbox =
+      Sandbox.builder().dar(Paths.get("./src/test/resources/ping-pong.dar")).build();
+
+  @ClassRule public static ExternalResource classRule = sandbox.getClassRule();
+
+  @Rule public ExternalResource rule = sandbox.getRule();
+
+  @Test
+  public void getSandboxPort() {
```

Comment:
I'm a bit confused what we would like to test here...
Maybe a rename? (I know it's a bit ugly, maybe you have a better idea)
```suggestion
  public void getSandboxPortReturnsTheDefaultPortIfUnspecified() {
```

Reply:
Yeah... so what happens here is that a port will be automatically assigned from the range `[6860, 6890]`, and we cannot specify it as of now.

Reply:
Ah ok